Userland/Libraries/LibGUI/TextDocument.cpp

```diff
@@ -343,6 +364,61 @@ String TextDocument::text() const
     return builder.to_string();
 }
 
+int TextDocument::text_len_behind_cursor(TextPosition const& cursor, TextPosition const& prev_word_break) const
+{
+    if (!cursor.is_valid())
+        return 0;
+
+    auto& line = this->line(cursor.line()); // lets see the text behind the cursor
```

Comment:
https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md#comments

Reply:
I ended up removing this comment. I think it may have made more sense when I first prototyping the fix, but I added some different, more accurate comments to the function